### ISSUE TITLE
Elemental Leatherworking recipes missing

### ISSUE THREAD

**Issue by Nimbux** [chromiecraft/chromiecraft#2620]
### What client do you play on?

enGB

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour

The NPC Brumn Winterhood (https://wowgaming.altervista.org/aowow/?npc=7869#teaches) does not offer leatherworking recipes even after completing the required quest (https://wowgaming.altervista.org/aowow/?quest=5146)

### Expected Blizzlike Behaviour

After completing "Elemental Leatherworking" quest (https://wowgaming.altervista.org/aowow/?quest=5146) you should have access to new recipes (https://wowgaming.altervista.org/aowow/?npc=7869#teaches) from said NPC.

### Steps to reproduce the problem

Level a horde character to 40 , learn leatherworking and skill it to a minimum of 225 , go to Arathi Highlands and talk to Brumn Winterhoof to learn Elemental Leatherworking.
After completing the quest he offers (https://wowgaming.altervista.org/aowow/?quest=5146) , try to learn the new recipes from him

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/24f4e08708c27b38ed46cb2333a53dfb4c484bc6

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Elexias** [comment]
You can only learn 3 recipes till 300+ (Outland) skill from him. That is whole point, watch minimum skill required to learn other recipes.

**Comment by Nimbux** [comment]
Yes, those 3 recipes can't be learned. Skilled LW to 300 and still can't learn them. The NPC in Arathi Highlands is bugged and the regular LW trainers in capital cities don't offer those recipes
